### ISSUE TITLE
Просмотр документов

### ISSUE THREAD

**Issue by metz-hei** [metz-hei/psb#114]
Как открывать пдфки и др форматы на устройстве. У Руслана Соловьева был кейс, когда док на мобиле открывался как превьюха и его можно было приближать и удалять и там же всякие кнопки на скачивание, печать и тд

**Comment by annabykovskaya** [comment]
С разработчиком РБ андроид (Коля Скудняков) сделали задачу, чтобы выпилить реализацию автоматического открытия дока в браузере при ошибке. Я добавила описание в задачу jira: PSBR-89369. Необходимо обсудить реализацию с дизайнерами.

**Comment by annabykovskaya** [comment]
Дока готова для финального ревью и дальнейшей публикации

**Comment by metz-hei** [comment]
Обсудить кнопку Открыть в браузере. Сколько за и сколько против этого решения. Не нравится, потому что нужна отдельная авторизация при переходе в веб

**Comment by annabykovskaya** [comment]
> Обсудить кнопку Открыть в браузере. Сколько за и сколько против этого решения. Не нравится, потому что нужна отдельная авторизация при переходе в веб

Нет, отдельная авторизация для этого не нужна. Документ открывается в просмоторщике ПДФ браузера. Но да, давай обсудим.

**Comment by annabykovskaya** [comment]
По остальным комментам получила ОК от ДС и разработчиков. В след. техдень постараюсь закрыть комменты с исследователями.

**Comment by annabykovskaya** [comment]
https://www.figma.com/design/lb9o7HAtQzt0R4D7cw3DQI/%F0%9F%93%9D-%F0%9F%8D%8E%F0%9F%A4%96-%D0%9F%D1%80%D0%BE%D1%81%D0%BC%D0%BE%D1%82%D1%80-%D0%B4%D0%BE%D0%BA%D1%83%D0%BC%D0%B5%D0%BD%D1%82%D0%BE%D0%B2?node-id=0-1

**Comment by annabykovskaya** [comment]
Вопросы по исследованиям закрыты. Задача готова к публикации.

**Comment by metz-hei** [comment]
Посмотрел. Все круто. Надо только хоум индикато везде вернуть

**Comment by metz-hei** [comment]
Готово. Ждем решения с гитом или не гитом
